src/controllers/enter.information.controller.ts

```diff
@@ -29,7 +30,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
   const company: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(session);
   const objectionId: string = retrieveFromObjectionSession(session, SESSION_OBJECTION_ID);
 
-  const reason: string = req.body.information;
+  const reason: string = removeNonPrintableChars(req.body.information);
```

Comment:
In theory might need to apply this to the only other text field where the user enters data - the 'full name / organisation name' field - but not mentioned in story so this PR is fine.